mcafee_atd/plugin.spec.yaml

```diff
@@ -643,3 +643,53 @@ actions:
         description: Returns true if found information about analyzer profiles
         type: boolean
         required: true
+  get_report:
+    title: Get Report
+    description: Download the analysis report file for given parameter
+    input:
+      id:
+        title: ID
+        description: Task ID or job ID or MD5 value for which will be prepared analysis report
+        type: string
+        required: true
+        example: 13
+      type_id:
+        title: Type ID
+        description: Type of given ID parameter, default value is MD5
+        type: string
+        required: false
+        default: MD5
+        example: TASK ID
+        enum:
+          - MD5
+          - TASK ID
+          - JOB ID
+      report_type:
+        title: Report Type
+        description: Type of file prepared analysis report
+        type: string
+        required: false
+        default: JSON
+        example: HTML
+        enum:
+          - JSON
+          - HTML
+          - TXT
+          - ZIP
+          - XML
+          - IOC
+          - STIX
+          - PDF
+          - SAMPLE
+    output:
+      file:
+        title: File
+        description: Prepared analysis report
+        type: bytes
+        required: true
+      report:
+        title: Report
+        description: Return report in JSON
+        type: object
```

Comment:
Is a custom type possible here?

Reply:
Custom type is different for every type of analysis sample (url, file, hash) so I think that we can't set here custom type.

Reply:
That makes sense 👍